### PR TITLE
Fix not correctly set confirmation code from Gmail in Spanish

### PR DESCRIPTION
**Pull request by sywhb**
* Forward text if html is not found in the email
* Fix not correctly set confirmation code from Gmail in Spanish

### CODE REVIEW

File: packages/inbound-email-handler/src/newsletter.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const getConfirmationCode = (subject: string): string | undefined => {
   if (matches) {
     // get the number code only
     // e.g. (#123456) => 123456
-    return matches[0].slice(2, -1)
```
Comment (jacksonh) [inline packages/inbound-email-handler/src/newsletter.ts:58]: I wonder if what we should do in the UX is just show the whole string. We'd have to change the UI a bit but the REGEX's and slicing seems a little brittle.

**Review by jacksonh — APPROVED** [review]

File: packages/inbound-email-handler/src/newsletter.ts (hunk 1 above)
Comment (sywhb) [reply packages/inbound-email-handler/src/newsletter.ts:58]: Yeah, I was thinking about it too.
One idea is we can always forward every confirmation email with an instruction or a link for setting the confirmation code in Omnivore
@jacksonh

Comment (jacksonh) [reply packages/inbound-email-handler/src/newsletter.ts:58]: Yeah, maybe creating an email with the info in it works best. The whole user interface is maybe just too confusing and brittle, but we could make a nice email.

**Review by jacksonh — APPROVED** [review]

File: packages/inbound-email-handler/src/newsletter.ts (hunk 1 above)
Comment (sywhb) [reply packages/inbound-email-handler/src/newsletter.ts:58]: I will create a ticket for this

**Pull request merged**